src/Core/Mews.Fiscalizations.Core/Model/Countries/Countries.cs

```diff
@@ -110,7 +110,7 @@ static Countries()
                 HolySee = NonEuropean("VA"),
                 Honduras = NonEuropean("HN"),
                 HongKong = NonEuropean("HK"),
-                Hungary = European("HU", "^(HU)?[0-9]{8}$"),
+                Hungary = European("HU", "^[0-9]{8}$"),
```

Comment:
The HU has to be allowed. So we can create a second regex and make Hungary use the second one. But generally, it's OK to provide the country code in TaxID.